devito/dse/rewriters.py

```diff
@@ -312,7 +285,7 @@ def _eliminate_inter_stencil_redundancies(self, cluster, template, **kwargs):
 
             # Build up the expression evaluating `alias`
             access = tuple(i.dim - i.lower for i in writeto)
-            expression = Eq(array[access], origin)
+            expression = Eq(array[access], origin.xreplace(subs))
```

Comment:
where does this xreplace come from?

Reply:
bug fix, otherwise missing some replacements

Reply:
But subs is initialized at `{}` line 260 and never updated.

Reply:
it is -- lines 297/298